src/vue/modules/create-atomic-swap-form/components/create-atomic-swap-form.vue

```diff
@@ -255,8 +298,16 @@ export default {
     async submit () {
       this.isFormSubmitting = true
       try {
+        // eslint-disable-next-line max-len
+        const isAmountMoreThanBalance = +this.form.amount > +this.accountBalance.balance
```

Comment:
comparison should be done with Math.util. JS has troubles with operations with some big numbers